comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -331,6 +332,15 @@ func (ia *inventoryagent) fetchSystemProbeMetadata() {
 	ia.data["feature_dynamic_instrumentation_enabled"] = sysProbeConf.GetBool("dynamic_instrumentation.enabled")
 }
 
+func (ia *inventoryagent) fetchPythonMetadata() {
```

Comment:
Wouldn't it make more sens to have this info in `inventorychecks` paylaod ?

Reply:
Will move it